src/common/YouTubeVideo.tsx

```diff
@@ -0,0 +1,64 @@
+import { AspectRatio, Box, Text } from "@chakra-ui/react";
+import Spinner from "./Spinner";
+
+export interface YoutubeVideo {
+  alt: string;
+  attribution: string;
+  caption: string;
+  youtubeId: String;
+}
+
+interface YoutubeVideoProps {
+  youTubeVideo: YoutubeVideo | undefined;
+}
+
+const YoutubeVideoEmbed = ({ youTubeVideo }: YoutubeVideoProps) => {
+  const { alt, attribution, caption, youtubeId } = youTubeVideo || {};
+  return (
+    <>
+      {youTubeVideo ? (
+        <Box as="figure">
+          <AspectRatio ratio={16 / 9}>
+            <iframe
+              // Avoid youtube cookie. rel=0 should limit related videos to our channel.
+              // Once we have translated videos we can try e.g. cc_lang_pref=fr
+              // but we'll need to check our codes match theirs.
+              title="welcome video"
+              src={`https://www.youtube-nocookie.com/embed/${
+                youtubeId ? youtubeId.trim() : ""
+              }?rel=0&cc_load_policy=1`}
+              allow="encrypted-media"
```

Comment:
Does this work via the versioner at the moment?

Reply:
I ran the versioner locally pointing to this review branch to test. The video played and there were no errors or warnings in the console.